search/controllers/api/__init__.py

```diff
@@ -334,4 +343,22 @@ def _parse_search_query(query: str) -> Dict[str, Any]:
             new_query_params[SEARCH_QUERY_FIELDS[field]] += " " + term
 
 
-    return new_query_params
+    return new_query_operators, new_query_params
+
+
+def _get_search_query(params: MultiDict, query_terms: List,
```

Comment:
Is this used?

Reply:
It appears that it is not, this was from an earlier FieldedSearchTerm implementation.

Reply:
Removed.